docs/administrative-requirements.md

```diff
@@ -43,15 +43,29 @@ For clients that are LHDI tenants:
 
 ### OBI Access Form
 
-Event Bus Consumers consuming a BIP-sourced event that is available on the Event Bus must follow the OBI Data Access Form process (aka the Corp DB Access Form).
+Event Bus Consumers consuming a BIP-sourced event that is available on the Event Bus must follow the OBI Data Access Form process (aka the Corp DB Application Access Form).
 
-If the same consumer wants to consume additional BIP-sourced events at a later time, another OBI Data Access Form must be submitted to cover the use of that additional data.
+**Who needs to submit an approval request?**
 
-Note that if _both_ the Event Bus and the Consumer are consuming an event for the first time, the Consumer must wait for the Event Bus to submit and get approval for its own OBI Data Access Form request before submitting its own request.
+Anyone who needs to access Corporate database data and has not been previously approved for that specific purpose must submit a request.
+
+Examples:
+- Any entity accessing Corporate data for the first time.
+- Any entity who has access to specific Corporate data but now needs access to a different data type.
+- Any entity who wants to use the Corporate data they have access to for a different purpose than what was previously approved. For example: VHA was previously approved to access rating data from Corporate through use of the VA Profile Rating data API. Now VHA would like to access direct deposit data from Corporate. This is a request for different data, so VHA would need to submit a request for approval to access this data.
+- Any entity accessing Corporate data through a third party. For example, those accessing data through:
```

Comment:
I think we can remove this section. Event Bus would probably be considered a third party, and I think we make it clear already that what consumers need to do to access the Corp DB data through events on the bus.

Reply:
All of this content on "Who needs to submit a request" was taken verbatim from the OBI instructions. I understand your recommendations are to tailor the bullets to Event Bus, but it might be worth modifying this bullet to say:

> Any entity accessing Corporate data through a third party (such as BIP)

... given that this section is BIP-specific

Reply:
Maybe "Any entity accessing Corporate data through a third party; for example, when a system consumes a BIP-sourced event from the Event Bus."?

And then remove the bullet points below...this is guidance for Event Bus usage only and we don't want to set ourselves up for advising on data consumption that systems may do through other third party systems.

Reply:
Ok, edited that bullet and formatted for consistency with previous examples.